### PR TITLE
New Admin Rest API

### LINKED ISSUE #14: Add feature to add probe to Admin
Admin allows adding objects to the Knowledge database. It is also needed to add the feature to add a probe to the database.

### PR DESCRIPTION
**Pull request by joseadp**
Closes #14

### CODE REVIEW

File: development/tma-admin-rest/src/main/java/eu/atmosphere/tmaf/admin/util/DatabaseManager.java [hunk 1]
```diff
@@ -0,0 +1,756 @@
...
+    private static final String SQL_SELECT_RESOURCE
+            = "SELECT * FROM Resource WHERE resourceId = ? ";
+    private static final String SQL_SELECT_CONFIGURATION
+            = "SELECT * FROM Configuration WHERE actionId = ? ";
+    private static final String SQL_SELECT_ACTUATOR
+            = "SELECT * FROM Actuator";
+    private static final String SQL_SELECT_PROBE
+            = "SELECT * FROM Probe";
+
+    /* TODO: Rui: find a better strategy */
```
Comment (nmsa) [inline development/tma-admin-rest/src/main/java/eu/atmosphere/tmaf/admin/util/DatabaseManager.java]: @rfrsilva please, check this out. Use a better strategy for passwd management, in line with other services.

File: development/tma-admin-rest/src/main/java/eu/atmosphere/tmaf/admin/util/DatabaseManager.java [hunk 2]
```diff
@@ -0,0 +1,756 @@
...
+            return null;
+        } finally {
+            close(con);
+        }
+    }
+
+    //Get Resources
+    public ArrayList<Resource> getResources(int actuatorId) {
+        int partnerId = actuatorId / 1000;
+        /* TODO: Paulo: why is this with LIKE? */
```
Comment (nmsa) [inline development/tma-admin-rest/src/main/java/eu/atmosphere/tmaf/admin/util/DatabaseManager.java]: @pgoncalvesgit  please, check this out.

File: development/tma-admin-rest/src/main/java/eu/atmosphere/tmaf/admin/util/DatabaseManager.java [hunk 3]
```diff
@@ -0,0 +1,756 @@
...
+            try {
+                con.close();
+            } catch (SQLException ex) {
+                LOGGER.error("[ATMOSPHERE] Error when closing the connection to the database.", ex);
+            }
+        }
+    }
+
+    /**
+     * TODO: This strategy is far from the best from a concurrency point of
```
Comment (nmsa) [inline development/tma-admin-rest/src/main/java/eu/atmosphere/tmaf/admin/util/DatabaseManager.java]: @joseadp this strategy assumes that we will never be inserting concurrently resources for the same partner. is this a valid assumption?

Comment (joseadp) [reply development/tma-admin-rest/src/main/java/eu/atmosphere/tmaf/admin/util/DatabaseManager.java]: @nmsa , yes. Up to now, the resources were static. Do we expect to have a high number of resources being added concurrently?

**Pull request merged**